pygly/alignment.py

```diff
@@ -776,7 +994,7 @@ def check_links_childlink_count(self, m, tg):
 
         return True
 
-    def subtree_leq(self, m, tg, root=True):
+    def subtree_leq(self, m, tg, root=True, **karg):
```

Comment:
Why is there a **karg here - it seems unused

Reply:
Short answer: yes, it is unused and it is to prevent "TypeError:  got an unexpected keyword argument"

This is a method in SubstructureSearchNonReducingEnd which inherited from SubstructureSearch.
This method gets overwritten. 
I believe the only difference between SubstructureSearchNonReducingEnd and SubstructureSearch is that SubstructureSearchNonReducingEnd additionally checks for reducing end link count. SubstructureSearchNonReducingEnd tried to use the rest machinery from SubstructureSearch.
The issue is that the subtree method in SubstructureSearch has additional parameters, and they are passed along during recursion.
Even if the glycan itself is not repeated, the parameter is there anyway.